### PR TITLE
contrib/bazel: new package

### PR DESCRIPTION
**Pull request by Gnarwhal**

### CODE REVIEW

File: contrib/bazel/patches/cbuild_flags.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     )
+ 
++    cbuild_flags_feature = feature(
++        name = "cbuild_flags",
++        enabled = True,
++        flag_sets = [
++            flag_set(
++                actions = all_compile_actions,
++                flag_groups = [flag_group(flags = [
++                    "-D_LARGEFILE64_SOURCE",
```
Comment (triallax) [inline contrib/bazel/patches/cbuild_flags.patch:15]: we're very unlikely to merge this with the problematic copying of the flags from cbuild  (will easily go out of sync, etc.)

i briefly looked up making the build respect our env vars and found https://github.com/bazelbuild/bazel/issues/1258, which was opened in 2016 and closed by a stalebot, so not very encouraging

Comment (Gnarwhal) [reply contrib/bazel/patches/cbuild_flags.patch:15]: Fair enough

Comment (q66) [reply contrib/bazel/patches/cbuild_flags.patch:15]: it could probably be done dynamically but not sure what that would look like

Comment (triallax) [reply contrib/bazel/patches/cbuild_flags.patch:15]: feel free to give it a try anyway :)

**Pull request closed without merging**